### PR TITLE
Fixed how parameterized test methods are sorted

### PR DESCRIPTION
**Pull request by Kropie**
Fixed how parameterized test methods are sorted to ensure that they use a natural sort.

### CODE REVIEW

**Comment by Kropie** [comment]
This PR will address issue #1465. See the attached screenshot to see the fix in action:
image

**Comment by jdneo** [comment]
Thank you @Kropie!  I'll take a look recently.

Meanwhile, could you fix the linting errors first? You can run `npm run lint` to find out where are the errors.

File: src/controller/utils.ts [hunk 1]
```diff
@@ -160,7 +160,7 @@ export function createTestItem(metaInfo: IJavaTestItem, parent?: TestItem): Test
     }
 
     if (metaInfo.testLevel === TestLevel.Invocation) {
-        item.sortText = metaInfo.id;
+        item.sortText = metaInfo.sortText;
```
Comment (jdneo) [inline src/controller/utils.ts:163]: Let's fallback to `metaInfo.id` when `metaInfo.sortText` is undefined

**Review by jdneo — CHANGES_REQUESTED** [review]
Looks good overall, just one small change request.

File: src/controller/utils.ts (hunk 1 above)
Comment (Kropie) [reply src/controller/utils.ts:163]: Sounds good! And I'll look into the linting errors asap

**Review by jdneo — APPROVED** [review]
Thank you @Kropie for your contribution!

**Pull request merged**